### ISSUE TITLE
Twig_Error_Runtime, after update symfony 3.4.4 => 3.4.5

### ISSUE THREAD

**Issue by ArmRes** [symfony/symfony#26384]
After update symfony (composer update), symfony update 3.4.4 to 3.4.5. And i get this error:
```
request.CRITICAL: Exception thrown when handling an exception 
(Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Object not found"). 
at C:\dev\project\vendor\symfony\symfony\src\Symfony\Bundle\TwigBundle\Resources\views\images\favicon.png.base64 line 1) {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Object not found\"). 
at C:\\dev\\project\\vendor\\symfony\\symfony\\src\\Symfony\\Bundle\\TwigBundle\\Resources\\views\\images\\favicon.png.base64:1, UnexpectedValueException(code: 0): Object not found at C:\\dev\\project\\vendor\\symfony\\symfony\\src\\Symfony\\Bridge\\Twig\\Extension\\ProfilerExtension.php:48)"} []
```

**Comment by jfcherng** [comment]
related https://github.com/twigphp/Twig/issues/2643.

**Comment by hktr92** [comment]
I did the upgrade, too. When I uninstalled the web profiler bundle, my apps (one symfony 3.4 and one 4.0) were functional.

**Comment by MakG10** [comment]
+1
I've got the the same error message as @hktr92 
![screenshot_20180303_074951](https://user-images.githubusercontent.com/11173162/36931513-760e9cd8-1eb7-11e8-9af5-183f68f66182.png)

**Comment by PhilETaylor** [comment]
Duplicate of 
https://github.com/twigphp/Twig/issues/2643
https://github.com/twigphp/Twig/issues/2627


My attempt of explaining and fixing it in a PR https://github.com/twigphp/Twig/pull/2645

**Comment by ArmRes** [comment]
Version Twig 2.4.6 fixes this problem, which was in 2.4.5
